aten/src/ATen/native/mps/operations/Copy.mm

```diff
@@ -180,7 +180,6 @@ void copy_cast_mps(at::Tensor& dst, const at::Tensor& src,
     // For View tensors, the storage offset can be bigger than what's being reported by nbytes
     src_total_size = at::detail::computeStorageNbytesContiguous(src.sizes(), src.element_size(), src.storage_offset());
   } else {
-    TORCH_INTERNAL_ASSERT(src_.strides() == dst_.strides());
```

Comment:
Why was this assert removed ?

Reply:
CPU fallback was crashing on this (in copy_kernel_mps code we already handle this case)